frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -43,7 +42,8 @@ pods:
         goal: RUNNING
         resource-set: server-resources
         cmd: >
-                ./bootstrap
+                ./bootstrap ;
+                pip install awscli
```

Comment:
will want to include this ahead of time somehow so that there aren't issues with offline clusters (nor pip flakes in general)
- standalone zip/build?
- docker image with awscli preinstalled?